harness/suite.go

```diff
@@ -203,6 +211,12 @@ func (s *Suite) Run() (err error) {
 		return err
 	}
 
+	reportDir := s.outputPath("reports")
+	if err := os.Mkdir(reportDir, 0777); err != nil {
+		return err
+	}
+	defer s.reporters.Output(reportDir)
```

Comment:
this is squashing the error. Since we have a named return error for this function already, this could be...

```
defer func() {
    if reportErr := s.reporters.Output(reportDir); reportErr != nil && err != nil {
        err = reportErr
    }
}()

Reply:
Pushed update with this change.